katello/katello-change-hostname

```diff
@@ -106,20 +106,27 @@ STDOUT.print("This script will modify your system. You will need to re-register
 response = yesno
 unless response
   STDOUT.puts "Hostname change aborted, no changes have been made to your system"
-  exit
+  exit(false)
 end
 
 unless options[:username] && options[:password]
   STDOUT.puts "Username and/or Password options are missing!"
   puts opt_parser
-  exit
+  exit(false)
 end
 
 if ARGV[0] && ARGV.count >= 1
   new_hostname = ARGV[0]
 else
   puts opt_parser
-  exit
+  exit(false)
+end
+
+hammer_ping = `hammer ping | grep FAIL`
+hammer_ping_failure = $?.success?
+if hammer_ping_failure
+  STDOUT.puts "All services are not currently running, please check 'hammer ping'"
+  exit(false)
```

Comment:
what about the case when `hammer ping` returns overall failure? To test this, stop your katello server. `hammer_ping_failure` is false in that case.